biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -266,13 +270,22 @@ subroutine disturbance_rates( site_in, bc_in)
 
```

Comment:
We should have had comments explaining what this is and in what situations it was used. We missed this the first time around.

Reply:
@ckoven added comments and generally improved readability in EDPatchDynamics, resolving